pkg/remotewrite/config.go

```diff
@@ -237,6 +237,19 @@ func parseEnvs(env map[string]string) (Config, error) {
 		c.Headers[k] = v
 	}
 
+	if headers, headersDefined := env["K6_PROMETHEUS_RW_HTTP_HEADERS"]; headersDefined {
+		if c.Headers == nil {
+			c.Headers = make(map[string]string)
+		}
+		for _, kvPair := range strings.Split(headers, ",") {
+			header := strings.Split(kvPair, ":")
+			if len(header) != 2 {
+				continue
```

Comment:
Maybe we could log a warning that something is ignored (this `contunue` happening), it would be better for the UX

Reply:
I thought we could just return an error and avoid adding the logger dependency.

Reply:
Yeah, also works since it attracts the user attention :+1: